### DISCUSSION TITLE
How do not display object above all others when clicked and selected?

### DISCUSSION THREAD

**Discussion by sgp86** [fabricjs/fabric.js#8501] [Q&A] [answered]
Hi, how to display an object in the current z-order, not above all others, when it is clicked and currently selected? Now, when the object is selected, it is always displayed above all others. I want to display it in the position (z-order) in which it really is. If it's underneath (when unchecked), then when I click it, I want it to be underneath too. If it's above others, let it be above others.

https://user-images.githubusercontent.com/25964230/206804924-533c074e-66db-486f-b143-f2fadfacb4d7.mp4

**Comment by ShaMan123** [comment, marked as answer]
Hopefully better docs will save you from such Q&A
`Canvas#preserveObjectStacking`
https://github.com/fabricjs/fabric.js/blob/41077213090b81c255d898a5dfbf94e2428e2317/src/canvas.class.ts#L349

**Reply by sgp86** [reply]
Cool, thanks, it works :)
